ga4gh/_protocol_definitions.py

```diff
@@ -746,6 +863,84 @@ def __init__(self, **kwargs):
         """
 
 
+class FeatureGroup(ProtocolElement):
```

Comment:
`FeatureGroup` is not a record type in the current Sequence Annotations code. Is this a concept unique to RNA, or is it referring to an older iteration of Sequence Annotations (where `FeatureGroup` was replaced with the `FeatureSet` container type)?